sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/Theme.kt

```diff
@@ -1,17 +1,66 @@
 package co.nimblehq.sample.compose.ui.theme
 
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.lightColors
-import androidx.compose.runtime.Composable
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 
 @Composable
 fun ComposeTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    MaterialTheme(
-        colors = lightColors(
-            primary = GreenCitrus
-        ),
-        content = content
-    )
+    val colors = if (darkTheme) {
+        DarkColorPalette
+    } else {
+        LightColorPalette
+    }
+    val dimensions = LocalAppDimensions.current
+    val styles = LocalAppStyles.current
+    val typography = LocalAppTypography.current
+    val shapes = LocalAppShapes.current
+
+    CompositionLocalProvider(
+        LocalAppColors provides colors,
+        LocalAppDimensions provides dimensions,
+        LocalAppStyles provides styles
```

Comment:
@luongvo What about `LocalAppTypoGraphy` & `LocalAppShape`, should they be added here too? 🙇

Reply:
@Tuubz After checking carefully, we only need to list `LocalAppColors provides colors` here to provide new colors value by `darkTheme` https://github.com/nimblehq/android-templates/blob/de566a4651372ecb658b84c946b043bcc316e1be/template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme/Theme.kt#L12-L16

The rest is unnecessary because we will use a single initial value without changing it.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/45b8205f525139e5abc7d3f57859b4212daca3d7